generator.rb

```diff
@@ -0,0 +1,111 @@
+def is_vowel(letter)
+  case letter
+  when "a", "e", "i", "o", "u"
+    return "true"
+  else
+    return "false"
+  end
+end
+
+def rev_str(strg)
+  aa = strg.split("")
+  rev = Array.new
+  aa.length.times do
+    rev << aa.pop
+  end
+  rev = rev.join
+  return rev
+end
+
+def check_length(word)
+  if word.length < 2
+    until word.length > 1
+      puts "Your word must be 2 or more characters."
+      word = gets.chomp
```

Comment:
I think this method would do exactly the same thing if you removed the `if` statement entirely:

```ruby
def check_length(word)
  until word.length > 1
    puts "Your word must be 2 or more characters."
    word = gets.chomp
  end
  return word
end
```